includes/admin/views/html-admin-settings.php

```diff
@@ -83,7 +83,13 @@
 				 * @since 1.8
 				 */
 				do_action( self::$setting_filter_prefix . '_tabs' );
-				?>
+
+				// Show link to New Reports page
+				$isReports = isset($_GET['page']) && $_GET['page'] === 'give-reports' ? true : false;
+				if ($isReports === true) { ?>
+				<a href="<?php echo admin_url('edit.php?post_type=give_forms&page=give-reports-v3') ?>" class="nav-tab"><?php esc_html_e( 'New Reports Page', 'give' ); ?></a>
```

Comment:
@henryholtgeerts for short HTML in between PHP we can use `sprintf` for better code formatting.

```
sprintf(
    '<a href="%1$s" class="nav-tab">%2$s</a>',
    admin_url('edit.php?post_type=give_forms&page=give-reports-v3'),
    esc_html__( 'New Reports Page', 'give' )
)
```

Reply:
thanks for the input, I applied your suggested changes with hotfixes on the reports development branch!

Reply:
See: https://github.com/impress-org/givewp/pull/4338/commits/5bbc514f7c6005564f4b9f4ac33a0cc33d7cadbe